sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -607,7 +607,8 @@ def __init__(self, source, test_bigquery_client=None, use_legacy_sql=True,
 
   def __enter__(self):
     self.client = BigQueryWrapper(client=self.test_bigquery_client)
-    self.client.create_temporary_dataset(self.executing_project)
+    self.client.create_temporary_dataset(
+        self.executing_project, self.source.table_reference)
```

Comment:
Can we pass location here instead of a table reference (so that we don't have to update this function when we support queries) ?